oscarlab/graphene
# [GLIBC] Separating the patch for supporting ld.gold

## Affected components


## Description of the changes (reasons and measures)

This PR separate the patch for fixing the building issue in glibc when `ld.gold` is the default linker in the system. The patch is introduced as a fix for #2. In this PR, we only apply the patch when `ld` is detected to be the same file as `ld.gold`.


## How to test this PR? (if applicable)

I check the compatibility with `ld.gold` by making `/usr/bin/ld` as a symbol link to `/usr/bin/ld.gold`.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/705)
